interfaces/builtin/usb_raw.go

```diff
@@ -0,0 +1,45 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const usbrawConnectedPlugAppArmor = `
+# Description: Allow raw access to all connected USB devices.
+# Reserved because this gives privileged access to the system.
+# Usage: reserved
+/dev/bus/usb/[0-9][0-9][0-9]/[0-9][0-9][0-9] rw,
+
+# Allow detection of usb devices. Leaks plugged in USB device info
+/sys/bus/usb/devices/ r,
+/sys/devices/pci**/usb[0-9]**/idVendor r,
+￼/sys/devices/pci**/usb[0-9]**/idProduct r,
+`
```

Comment:
@teknoraver Could you help to add below apparmor rules to this usb-raw interfaces. This would be help for Intel Realsense applications. See my pull request for realsense interface https://github.com/snapcore/snapd/pull/2236#pullrequestreview-6607126 . If these rules being intergrated into usb-raw interfaces. Then I can leverage usb-raw interface directly and will not need to add another interface for realsense applications.

/sys/devices/pci**/usb*/**/busnum r,
/sys/devices/pci**/usb*/devnum r,
/sys/devices/pci**/usb*/**/devnum r,
/sys/devices/pci**/usb*/**/modalias r,
/sys/devices/pci**/usb*/**/bInterfaceNumber r,